src/applications/disability-benefits/686c-674/tests/e2e/00-spouse-child-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,197 @@
+const Auth = require('platform/testing/e2e/auth');
+const E2eHelpers = require('platform/testing/e2e/helpers');
+const Timeouts = require('platform/testing/e2e/timeouts');
+const manifest = require('../../manifest.json');
+const testData = require('./686-test-data.json');
+
+import * as TestHelpers from './test-helpers';
+
+const runTest = E2eHelpers.createE2eTest(client => {
+  const token = Auth.getUserToken();
+  TestHelpers.initApplicationSubmitMock(token);
+  // Login
+  Auth.logIn(
+    token,
+    client,
+    '/disability-benefits/new-686/introduction',
+    3,
+  ).waitForElementVisible('.process.schemaform-process', Timeouts.verySlow);
+  client.click('.usa-button-primary.va-button-primary.schemaform-start-button');
+
+  // select options
+  E2eHelpers.expectLocation(client, '/686-options-selection');
+  client.axeCheck('.main');
+  TestHelpers.select686Options(
+    client,
+    ['addChild', 'addSpouse'],
+    testData.data,
+  );
+  client.click('button[id="4-continueButton"]');
+
+  // veteran information
+  E2eHelpers.expectLocation(client, '/veteran-information');
+  client.axeCheck('.main');
+  TestHelpers.fillVeteranData(client, testData.data);
+  client.click('button[id="4-continueButton"]');
+
+  // veteran address
+  E2eHelpers.expectLocation(client, '/veteran-address');
+  client.axeCheck('.main');
+  TestHelpers.fillVeteranDomesticAddress(client, testData.data);
+  client.click('button[id="4-continueButton"]');
+
+  // child information
+  E2eHelpers.expectLocation(client, '/add-child');
+  client.axeCheck('.main');
+  TestHelpers.fillChildNameInformation(client, testData.data, 0);
+  client.click('.va-growable button.usa-button-secondary.va-growable-add-btn');
+  TestHelpers.fillChildNameInformation(client, testData.data, 1);
+  client.click('button[id="4-continueButton"]');
+
+  // child 1 place of birth and status
+  E2eHelpers.expectLocation(client, '/add-child/0');
+  client.axeCheck('.main');
+  TestHelpers.fillChildPlaceOfBirthAndStatusInformation(client, testData.data);
+  client.click('button[id="4-continueButton"]');
+
+  // child 1 current living location
+  E2eHelpers.expectLocation(client, '/add-child/0/additional-information');
+  client.waitForElementVisible(
+    '#root_doesChildLiveWithYou-label',
+    Timeouts.normal,
+  );
+  client.axeCheck('.main');
+  client.pause(Timeouts.normal);
```

Comment:
Not really a fan of using these `.pause` lines, but Nightwatch doesn't seem to be able to find radio buttons very well, even when using `client.waitForElementVisible()` and several different selectors. The only way I could get the test to actually find the radio buttons was to pause before each one. If there is a better way to approach this please please tell me.